data/governments.txt

```diff
@@ -395,6 +395,8 @@ government "Hai"
 		"Merchant" .01
 		"Bounty Hunter that Attacks Hai" -.01
 		"Elenctic Commune" .1
+	restricted
```

Comment:
you know my opinion on this?
so you better give a reason for this

Reply:
Its totally unintended that the hai attack the unfettered